doc/start/quick-client.rst

```diff
@@ -0,0 +1,33 @@
+=======================
+Basic Ceph Client Setup
+=======================
+Client machines need some basic configuration in order to interact with
+a cluster. This document describes how to configure a client machine
+for cluster interaction.
+
+Note that most client machines only need the `ceph-common` package and
+its dependencies installed.
+
+Config File Setup
+=================
+Client machines can generally get away with a smaller config file than
+a full-fledged cluster member. To generate a minimal config file, log
+into a machine that is already a cluster member and run::
```

Comment:
I don't know what "is already cluster member" means here. Can a machine be part of cluster without having client configured on it or having a Ceph daemon deployed on it?

Reply:
Good point. I updated the document and hopefully clarified that a bit.